azimuth_caas_operator/utils/ansible_runner.py

```diff
@@ -90,7 +90,7 @@ def get_env_configmap(
 ):
     extraVars = dict(cluster_type_spec.extraVars, **cluster.spec.extraVars)
     extraVars["cluster_name"] = cluster.metadata.name
-    extraVars["cluster_id"] = cluster.metadata.uid
+    extraVars["cluster_id"] = f"{cluster.metadata.namespace}--{cluster.metadata.name}"
```

Comment:
I don't think this is going to be backwards compatible...

We need a different way to do this I think. Maybe we generate the cluster_id within Azimuth, and inject that into extraVars? We can change this to use the uid only if cluster_id is missing, and eventually we remove that logic? Not 100% sure.

Reply:
Can we use an underscore instead of `--`? Underscore is not valid in Kubernetes names.